BedrockServer.cpp

```diff
@@ -1404,16 +1403,20 @@ bool BedrockServer::shutdownComplete() {
 }
 
 void BedrockServer::prePoll(fd_map& fdm) {
-    STCPServer::prePoll(fdm);
+    // Add all our ports. There are no sockets directly managed here.
+    if (_commandPort) {
+        SFDset(fdm, _commandPort->s, SREADEVTS);
+    }
+    if (_controlPort) {
+        SFDset(fdm, _controlPort->s, SREADEVTS);
+    }
+    for (const auto& p : _portPluginMap) {
+        SFDset(fdm, p.first->s, SREADEVTS);
+    }
 }
 
 void BedrockServer::postPoll(fd_map& fdm, uint64_t& nextActivity) {
-    // Let the base class do its thing. We lock around this because we allow worker threads to modify the sockets (by
-    // writing to them, but this can truncate send buffers).
-    {
-        STCPServer::postPoll(fdm);
```

Comment:
Same as above, since we only handle reading new sockets off ports, the base class doesn't do anything useful anymore.